<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PantallaUsuarios</class>
 <widget class="QWidget" name="PantallaUsuarios">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>509</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>628</width>
    <height>509</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/fondo/imagen-llamativa-cintas-sobre-fondo-oscuro-generacion-ai_724548-22165.jpg);</string>
  </property>
  <widget class="QTableWidget" name="tablaUsuarios">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>221</y>
     <width>601</width>
     <height>271</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>371</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnAgregar">
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);
border:2px solid #ffffff;
}
QPushButton:hover{
border-radius:10px;
font:12pt&quot;Arial&quot;;
color: rgb(255,255,255);
border:2px solid rgb(120,0,171);
  background-color: rgb(170, 85, 0);
}
</string>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEditar">
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);
border:2px solid #ffffff;
}
QPushButton:hover{
border-radius:10px;
font:12pt&quot;Arial&quot;;
color: rgb(255,255,255);
border:2px solid rgb(120,0,171);
  background-color: rgb(170, 85, 0);
}

</string>
      </property>
      <property name="text">
       <string>Editar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEliminar">
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);
border:2px solid #ffffff;
}
QPushButton:hover{
border-radius:10px;
font:12pt&quot;Arial&quot;;
color: rgb(255,255,255);
border:2px solid rgb(120,0,171);
  background-color: rgb(170, 85, 0);
}
</string>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancelar">
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color: rgb(127, 0, 190);
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);
border:2px solid #ffffff;
}
QPushButton:hover{
border-radius:10px;
font:12pt&quot;Arial&quot;;
color: rgb(255,255,255);
border:2px solid rgb(120,0,171);
  background-color: rgb(170, 85, 0);
}
</string>
      </property>
      <property name="text">
       <string>Cancelar/Reintentar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>171</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/fondo/perfil-de-usuario (1).png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>371</width>
     <height>96</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lblUsuarios">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{	
color:rgb(255,255,254);
font: 12pt&quot;Arial&quot;;
background-color: rgb(170, 85, 0);;
}</string>
      </property>
      <property name="text">
       <string>Usuario</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtUsuario">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
/*background-color:rgba(0,0,0);
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);
border:2px solid rgb(255,255,255);*/
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);


}

QLineEdit:hover{
/*background-color:rgba(0,0,0);
background-color: :rgba(12,12,12);
border-radius:10px;
font:12pt&quot;Arial&quot;;
border:2px solid rgb(120,0,171);*/

border-radius:10px;
font:12pt&quot;Arial&quot;;
/*color: rgb(255,255,255);*/
border:2px solid rgb(120,0,171);
  background-color: rgb(170, 85, 0);
}
QLineEdit:focus{
background-color:rgba(0,0,0);
border-radius:10px;
font:12pt&quot;Arial&quot;;
border:2px solid rgb(120,0,171);
}
</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblContrasenia">
      <property name="styleSheet">
       <string notr="true">QLabel{	
color:rgb(255,255,254);
font: 12pt&quot;Arial&quot;;
background-color: rgb(170, 85, 0);;
}</string>
      </property>
      <property name="text">
       <string>Contrase√±a</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtContrasenia">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
/*background-color:rgba(0,0,0);
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);
border:2px solid rgb(255,255,255);*/
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);


}

QLineEdit:hover{
/*background-color:rgba(0,0,0);
background-color: :rgba(12,12,12);
border-radius:10px;
font:12pt&quot;Arial&quot;;
border:2px solid rgb(120,0,171);*/

border-radius:10px;
font:12pt&quot;Arial&quot;;
/*color: rgb(255,255,255);*/
border:2px solid rgb(120,0,171);
  background-color: rgb(170, 85, 0);
}
QLineEdit:focus{
background-color:rgba(0,0,0);
border-radius:10px;
font:12pt&quot;Arial&quot;;
border:2px solid rgb(120,0,171);
}
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="recursos/image.qrc"/>
 </resources>
 <connections/>
</ui>
